kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/filtered-resources/kjars-sources/definition-project/src/main/resources/META-INF/kie-deployment-descriptor.xml

```diff
@@ -50,5 +50,6 @@
     <configurations/>
     <required-roles/>
     <remoteable-classes>
+        <remoteable-class>org.jbpm.document.service.impl.DocumentImpl</remoteable-class>
```

Comment:
is this really needed?

Reply:
no, I forgot to undo, I was trying to avoid the jaxb marshaller modificaiton